web/frontend/src/components/slam/2d-renderer.svelte

```diff
@@ -29,7 +29,7 @@ const pointsRenderOrder = 3;
 const axesHelperRenderOrder = 2;
 const gridHelperRenderOrder = 1;
 
-const cameraScale = 12.5;
+const defaultPointScale = 13.2;
```

Comment:
Whats the significance of the value change?

Reply:
This bumps up the default point size on the 2d slam map. We've noticed on a few screens that points are still too small to be easily seen, and we still have a little bit of room on all screens we've observed before they start to bump into each other